web/styles/zulip.css

```diff
@@ -2868,12 +2868,12 @@ select.invite-as {
         color: var(--color-navbar-icon);
     }
 
-    .zulip-icon-gear {
+    .zulip-icon-gear,
+    .zulip-icon-user-list {
         font-size: 18px;
     }
 
-    .zulip-icon-help-bigger,
-    .zulip-icon-triple-users {
+    .zulip-icon-help-bigger {
         font-size: 20px;
```

Comment:
Is there a discussion for using 18px font-size here? 20px looks better to me.

Reply:
changed to 20 px.